### ISSUE TITLE
M-Backspace no longer works

### ISSUE THREAD

**Issue by Tcc100** [tmux/tmux#1979]
### Issue description

08b07b1 breaks Meta+Backspace, i.e. delete word backwards.
The fetch from options removes the modifier.

It can be fixed using the following patch, however, there probably is a better way:

```
diff --git a/input-keys.c b/input-keys.c
--- a/input-keys.c
+++ b/input-keys.c
@@ -178,9 +178,9 @@ input_key(struct window_pane *wp, key_code key, struct mouse_event *m)
 
        /* Is this backspace? */
        if ((key & KEYC_MASK_KEY) == KEYC_BSPACE) {
-               key = options_get_number(global_options, "backspace");
+               key = options_get_number(global_options, "backspace") | (key & ~KEYC_MASK_KEY);
                if (key >= 0x7f)
-                       key = '\177';
+                       key = '\177' | (key & ~KEYC_MASK_KEY);
        }
 
        /*
```

**Comment by nicm** [comment]
This is already fixed in 4bc445f.

**Comment by Tcc100** [comment]
Ah yes, now I see, then never mind.
I confused my remotes and missed the last few commits.

I just tested the correct HEAD and it works.

Sorry and thank you for the fast response.
